docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -8829,6 +8829,222 @@ with the character set specified by _char-set-name_.
 SPACE(3)
 ```
 
+<<<
+[[split_part_function]]
+== SPLIT_PART Function
+
+The SPLIT_PART function returns the n^th^ substring that is split on the defined `_delimiter_` and `_filed-position_`.
```

Comment:
I think you mean "field_position" rather than "filed_position" (here, and below)

Reply:
Oops, thanks Dave, I've corrected 😁